src/vs/platform/shell/node/shellEnv.ts

```diff
@@ -139,7 +139,7 @@ async function doResolveShellEnv(logService: ILogService, token: CancellationTok
 		// Older versions of PowerShell removes double quotes sometimes
 		// so we use "double single quotes" which is how you escape single
 		// quotes inside of a single quoted string.
-		command = `Write-Output '${mark}'; [System.Environment]::GetEnvironmentVariables() | ConvertTo-Json -Compress; Write-Output '${mark}'`;
+		command = `chcp 65001; Write-Output '${mark}'; [System.Environment]::GetEnvironmentVariables() | ConvertTo-Json -Compress; Write-Output '${mark}'`;
```

Comment:
...wait does this run on macOS & Linux as well with PowerShell 7+? If so `chcp` is not available there and this will break.

Reply:
Oops, I tested `chcp` and it doesn't work on linux.
I noticed you already fixed it, thanks!